### PR TITLE
Remove stuff from common-adapters, factor out some platform differences

### PR DESCRIPTION
**Pull request by zanderz**
Still removing/moving some other stuff too

### CODE REVIEW

**Comment by zanderz** [comment]
I've reached a stopping point on this I think. Is Jenkins PR just a flake?

File: shared/chat/conversation/input-area/normal/platform-input.desktop.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import * as React from 'react'
 import * as Kb from '../../../../common-adapters'
 import * as Styles from '../../../../styles'
+import animationData from '../../../../common-adapters/animation-data.json'
```
Comment (chrisnojima) [inline shared/chat/conversation/input-area/normal/platform-input.desktop.js]: this should go back. we're going to have more animation stuff in the future

File: shared/chat/conversation/input-area/normal/platform-input.native.js [hunk 2]
```diff
@@ -12,14 +12,8 @@ import {
...
 } from '../../../../common-adapters'
-import {
-  collapseStyles,
-  globalMargins,
-  globalStyles,
-  globalColors,
-  platformStyles,
-  styleSheetCreate,
-} from '../../../../styles'
+import animationData from '../../../../common-adapters/animation-data.json'
```
Comment (chrisnojima) [inline shared/chat/conversation/input-area/normal/platform-input.native.js]: same here

File: shared/chat/conversation/input-area/normal/platform-input.desktop.js (hunk 1 above)
Comment (zanderz) [reply shared/chat/conversation/input-area/normal/platform-input.desktop.js]: ok

File: shared/chat/conversation/input-area/normal/platform-input.native.js (hunk 2 above)
Comment (zanderz) [reply shared/chat/conversation/input-area/normal/platform-input.native.js]: ok

**Comment by zanderz** [comment]
@nojima is this ok if I resolve conflicts?

**Pull request merged**